docs/devguide/building/index.rst

```diff
@@ -605,7 +605,7 @@ These are all the variables that are exclusive to building OpenDDS with CMake:
 .. cmake:var:: OPENDDS_ACE_TAO_KIND
 
   The default is ``ace7tao3`` for :ref:`ACE 7/TAO 3 <ace7tao3>`.
-  Use ``ace6tao2`` to get :ref:`ACE 6/TAO 2 <ace6tao2>`.
+  See :ghfile:`acetao.ini` for other versions of ACE/TAO.
```

Comment:
```suggestion
  See :ref:`here <deps-ace-tao>` for other versions of ACE/TAO.
```

A little nicer than the raw ini file and a little more info, like what C++ standards they require.